### DISCUSSION TITLE
Rename ALTERNATE_ID to ID_TO_LOG_OUT or ID_FOR_OUTPUT or something like that

### DISCUSSION THREAD

**Discussion by danisyellis** [indeedeng/starfish#85] [Ideas]
I think this change will make it more obvious what that ID is for.

As part of this change, update the README in any way necessary to make it as understandable as possible that this is controlled by the CSV and the number you put in the ENV variable, not in the code.

**Comment by kevindigo** [comment]
Between those 2 options, I prefer ID_FOR_OUTPUT.

**Comment by danisyellis** [comment]
Woo-hoo, I just converted this issue to a discussion, because that's a thing we can do now.
I'm curious if anyone will have a better idea of what to name this variable.
